storage/storage.go

```diff
@@ -424,14 +513,59 @@ func (storage DBStorage) WriteReportForCluster(
 		return err
 	}
 
+	deleteQuery := "DELETE FROM rule WHERE org_id = $1 AND cluster_id = $2;"
+	_, err = tx.Exec(deleteQuery, orgID, clusterName)
+	if err != nil {
+		log.Err(err).Msgf("Unable to remove previous cluster reports (org: %v, cluster: %v)", orgID, clusterName)
+		_ = tx.Rollback()
+		return err
+	}
+
+	// Perform the report upsert.
+	reportedAtTime := time.Now()
+
+	for _, rule := range rules {
+		ruleID := fmt.Sprintf("%v|%v", rule.Module, rule.ErrorKey)
```

Comment:
Why do you use ruleID combined with ruleErrorKey by '|' sign? I think it's gonna confuse us a lot and postgres has no problems with composite primary keys.

Reply:
It can't cause problems since it's python module's name and  '|' can't be used there, but it looks weird. But consider it as a nitpick :)

Reply:
Because there can be different reports with same `Module`, but different `ErrorKey`, thus only combination of this two values might be perceive as unique key

Reply:
Yes, and this is what composite primary keys are for